lib/xcode/install.rb

```diff
@@ -141,21 +141,28 @@ def current_symlink
       File.symlink?(SYMLINK_PATH) ? SYMLINK_PATH : nil
     end
 
-    def download(version, progress, url = nil, progress_block = nil, retry_download_count = 3)
+    def download(version, progress, url = nil, local_url = nil, progress_block = nil, retry_download_count = 3)
```

Comment:
Could you explain why we couldn't pass the local URL as the `url` param? Why did we need a new parameter for it? It's not clear to me 😬